### DISCUSSION TITLE
Is it possible to use this bundle without the AssetMapper?

### DISCUSSION THREAD

**Discussion by bsprengelmeijer** [Spomky-Labs/pwa-bundle#228] [Q&A]
I'm using symfony/ux-vue, so I need webpack-encore to build the assets, but using webpack together with the asset mapper doesn't work.

**Comment by tacman** [comment]
Alternatively, re-write your view code to not use .vue files.

> If you do still want to use Vue with AssetMapper, you can do so by avoiding the .vue file format. For example, https://github.com/symfony/ux/blob/2.x/ux.symfony.com/assets/vue/controllers/PackageSearch.js.

That way, you can use AssetMapper, which rocks.  Let us know if that works for you, I've been tempted to switch some things to vue but at this point I can't imagine giving up AssetMapper.

**Comment by Spomky** [comment]
Hi @bsprengelmeijer,

Indeed, Asset Mapper and Webpack lead to undesired behaviors.
I tried to migrate the components based on SF UX React, but it failed and I didn't try again.
Another solution could be to compile the files directly before calling Asset Mapper. The [Tailwind bundle](https://symfony.com/bundles/TailwindBundle/current/index.html) does it this way.

1. Remove Webpack Encore and related files
2. Use Vue CLI to compile your `.vue` files. The target folder should be in `assets/`
3. Import compiled `css` and `js` files from your templates or `app.js`
